app/airq/commands/quality.py

```diff
@@ -51,28 +54,23 @@ def _get_message(self, zipcode: Zipcode) -> typing.List[str]:
                 f" (AQI {aqi})" if aqi else "",
             )
         )
-        message.append("")
 
         has_zipcode = self.client.zipcode_id is not None
         was_updated = self.client.update_subscription(zipcode)
         if not self.client.is_enabled_for_alerts:
+            message.append("")
             message.append(
                 'Alerting is disabled. Text "Y" to re-enable alerts when air quality changes.'
             )
         elif was_updated:
-            zipcode_updated_message = (
-                "You'll receive timely texts when AQI in your area changes based on PurpleAir data. "
-                'Text Menu ("M") for more features including recommendations, or end alerts by texting "E".'
-            )
             if has_zipcode:
-                message.append(zipcode_updated_message)
-            else:
-                message.append(f"Thanks for texting Hazebot! {zipcode_updated_message}")
+                # Zipcode changed.
                 message.append("")
-                message.append(
-                    "Save this contact (most call me Hazebot) and text your zipcode anytime for an AQI update."
-                )
+                message.append(f"You are now receiving alerts for {zipcode.zipcode}.")
+            else:
+                tasks.send_intro_message.apply_async((self.client.id,), countdown=30)
```

Comment:
Don't we want to only send this when a client is first created?

Reply:
Yeah I think so

Reply:
That's what's happening here: we send it the first time they choose a zipcode, which isn't necessarily exactly when they're created but is likely the first time we want to send them this